python/cudf/cudf/core/multiindex.py

```diff
@@ -1458,7 +1458,7 @@ def from_pandas(cls, multiindex, nan_as_null=None):
         )
         return cls.from_frame(df, names=multiindex.names)
 
-    @cached_property
+    @cached_property  # type: ignore
     @_cudf_nvtx_annotate
     def is_unique(self):
```

Comment:
Instead of ignoring the type:
```suggestion
    def is_unique(self) -> bool:
```

Reply:
As above, this doesn't help with the "decorated property not supported" error